infer/src/IR/AccessPath.mli

```diff
@@ -7,7 +7,7 @@
 
 open! IStd
 
-(** Module for naming heap locations via the path used to access them (e.g., x.f.g, y[a].b) *)
+(** Module for naming heap locations via the path used to access them (e.g., [x.f.g, y[a].b]) *)
```

Comment:
Unless I'm mistaken, if brackets are added around something containing brackets, the inner brackets should be escaped (or else ocamldoc will not display them).